pdal/Options.hpp

```diff
@@ -154,12 +154,12 @@ class PDAL_DLL Options
         {
             StringList l = getValues(k);
             std::string vs;
-            for (auto vi = l.begin(); vi != l.end(); ++vi)
-            {
-               if (vi != l.begin())
-                   vs += ", ";
-               vs += *vi;
-            }
+//             for (auto vi = l.begin(); vi != l.end(); ++vi)
```

Comment:
If you don't want this, I'd suggest deleting rather than commenting -- there's always source control  history -- just came from a project with mountains of commented code and it's terrible.